### PR TITLE
docs: how to copy staging db to local development server

### PR DESCRIPTION
**Pull request by erictapen**
Until https://github.com/Nix-Security-WG/nix-security-tracker/issues/198 is fixed

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -129,12 +129,15 @@ hivemind
...
+In order to start over, delete the database and recreate it, then restore it from staging, and (just in case the dump is behind the code) run migrations.
+
+> [!NOTE]
+> For now this requires SSH access to the staging environment. In the future you will be able to download a publicly available dump.
 
 ```bash
 dropdb nix-security-tracker
 sudo -u postgres createdb -O nix-security-tracker nix-security-tracker
-pg_restore -O -d nix-security-tracker -v < dump
+ssh sectracker.nixpkgs.lahfa.xyz "sudo -u postgres pg_dump --create web-security-tracker | zstd" | zstdcat | sed 's|web-security-tracker|nix-security-tracker|g' | pv | psql
```
Comment (fricklerhandwerk) [inline CONTRIBUTING.md:140]: Please make that an *addition* so we don't lose track of the direct-from-dump workflow.

Comment (erictapen) [reply CONTRIBUTING.md]: done

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -138,6 +138,12 @@ pg_restore -O -d nix-security-tracker -v < dump
 manage migrate
 ```
 
+If you have SSH access to the staging environment, you can instead dump and restore a fresh dump:
```
Comment (fricklerhandwerk) [inline CONTRIBUTING.md:141]:
```suggestion
If you have SSH access to the staging environment, you can instead dump and restore the latest state directly:
```

**Review by fricklerhandwerk — APPROVED** [review]
Thanks!

**Pull request merged**